terraform/modules/enclave/paas-config/prometheus.conf.tpl

```diff
@@ -11,3 +18,9 @@ scrape_configs:
     file_sd_configs:
       - files: ['/etc/prometheus/targets/*.json']
         refresh_interval: 30s
+  - job_name: alertmanagers
```

Comment:
These job doesn't have exactly the same as our ECS file (here alertmanager has been made plural). This means that our alerts we've written for alertmanager will not fire - https://github.com/alphagov/prometheus-aws-configuration-beta/blob/master/terraform/projects/app-ecs-services/config/alerts/observe-alerts.yml

Reply:
Good catch this very important as we would have broken this in the worst way possible